tests/test_other.py

```diff
@@ -6118,12 +6118,12 @@ def test_massive_alloc(self):
     ''')
     run_process([PYTHON, EMCC, 'main.cpp', '-s', 'ALLOW_MEMORY_GROWTH=1', '-s', 'WASM=0'])
     # just care about message regarding allocating over 1GB of memory
-    output = run_js('a.out.js', stderr=PIPE, full_output=True, engine=SPIDERMONKEY_ENGINE)
+    output = run_js('a.out.js', stderr=PIPE, full_output=True, engine=V8_ENGINE)
     self.assertContained('''Warning: Enlarging memory arrays, this is not fast! 16777216,1543503872\n''', output)
     print('wasm')
     run_process([PYTHON, EMCC, 'main.cpp', '-s', 'ALLOW_MEMORY_GROWTH=1'])
     # no message about growth, just check return code
-    run_js('a.out.js', stderr=PIPE, full_output=True, engine=SPIDERMONKEY_ENGINE)
+    run_js('a.out.js', stderr=PIPE, full_output=True, engine=V8_ENGINE)
```

Comment:
Can node still not do these things?

Reply:
Good point, yeah, node can do them these days. So we could just remove all these special cases. That would be simpler, for a small loss of more testing on a single shell. Maybe that's better though, what do you think?

Reply:
I don't think we can really consider d8 and node to be different engines since they are based on the same codebase.    I'm tempted to limit emscripten to node LTS + browsers only to keep things simple in the testing matrix.